.github/workflows/ci.yml

```diff
@@ -33,45 +33,28 @@ jobs:
     - uses: risc0/risc0/.github/actions/sccache@release-0.19
     - name: Install cargo-binstall
       uses: cargo-bins/cargo-binstall@v1.6.4
-    - run: cargo binstall -y --force cargo-risczero@${{ env.RISC0_VERSION }}
-    - run: cargo risczero install --version $RISC0_TOOLCHAIN_VERSION
+    - run: cargo binstall -y --force cargo-risczero
+    - run: cargo risczero install
     - run: cargo build -p risc0-guest
 
   build-sp1:
     name: Build sp1
     runs-on: ubuntu-latest
     timeout-minutes: 60
     steps:
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v4
       - uses: actions-rs/toolchain@v1
         with:
           profile: minimal
           toolchain: nightly-2024-01-25
           override: true
           components: clippy, rustfmt, rust-src
-      - run: curl -L https://sp1.succinct.xyz | bash
-      - name: Use sp1up
+      - name: Setup and build
         run: |
-          source /home/runner/.bashrc
-          sp1up
+          curl -L https://sp1.succinct.xyz | bash
+          /home/runner/.config/.sp1/bin/sp1up
+          cargo build -p raiko-host --features succinct
         shell: bash
-      - run: cargo build -p raiko-host --features succinct
-
-  test:
```

Comment:
why no test? is it placed somewhere else?

Reply:
i just removed it because it's just doing a `cargo check` which `cargo build` already does

Reply:
[2e4a832](https://github.com/taikoxyz/raiko/pull/56/commits/2e4a832232506bf5f8104fd919bfa51d681892c5)